src/broadcast/mpris/mprisplayer.cpp

```diff
@@ -416,4 +433,39 @@ void MprisPlayer::setRate(double value) {
     }
 }
 
+void MprisPlayer::slotCoverArtFound(const QObject *requestor,
+                                    const CoverInfoRelative &info,
+                                    QPixmap pixmap,
+                                    bool fromCache) {
+
+    if (!pixmap.isNull()) {
+        QImage coverImage = pixmap.toImage();
+        QString imagePath = QDir::tempPath() + "/" + qHash(m_currentMetadata.title)
+                + "jpg";
+        bool success = coverImage.save(imagePath,"JPG");
```

Comment:
When is the cover image deleted?

Reply:
It isn't, I write it in the /tmp/ folder so the OS takes care of it. I could keep record and delete all images but I don't know if it's worth the time.